spec/std/array_spec.cr

```diff
@@ -1060,11 +1060,28 @@ describe "Array" do
     end
   end
 
-  it "does unshift" do
-    a = [2, 3]
-    expected = [1, 2, 3]
-    a.unshift(1).should eq(expected)
-    a.should eq(expected)
+  describe "unshift" do
+    it "adds one element to the beginning of the array" do
+      a = [1, 2]
+      a.unshift(3)
+      a.should eq [3, 1, 2]
+    end
+
+    it "returns the array when adding one element" do
+      a = [1, 2]
+      a.unshift(3).should eq [3, 1, 2]
```

Comment:
Please use `should be(a)`. We want to check that the array returned is exactly the same, not maybe another array with the same contents. In fact, I would just make it a single spec. (same goes with the next two specs)